src/multivariate/composite.jl

```diff
@@ -0,0 +1,203 @@
+# Composite Multivariate distribution
+
+## Generic Composite multivariate distribution class
+
+module CompositeDistributions
+
+if !isdefined(:Distributions) using Distributions end
```

Comment:
This line seems like it was relevant to a version of this code that lived outside of Distributions, but not a version that lives inside of Distributions.